### PR TITLE
Add Christmas theme

### PR DESCRIPTION
**Pull request by ronalduQualabs**
A festive Christmas theme for the player with cozy red and green tones, twinkling lights, and a warm holiday vibe—perfect for spreading seasonal cheer!

image

### CODE REVIEW

File: themes/x-mas/template.html [hunk 1]
```diff
@@ -0,0 +1,2222 @@
...
+      align-items: center;
+      justify-items: center;
+      height: 70%;
+      width: 20%;
+      margin-top: -1rem;
+      border-radius: 9999px;
+
+      &:hover {
+      }
+
```
Comment (luwes) [inline themes/x-mas/template.html:92]:
```suggestion
```

File: themes/x-mas/template.html [hunk 2]
```diff
@@ -0,0 +1,2222 @@
...
+      svg {
+        height: 100%;
+        filter: drop-shadow(0px 0px 36px #00000069);
+      }
```
Comment (luwes) [inline themes/x-mas/template.html:96]: just noting CSS nesting is relatively new https://caniuse.com/?search=css%20nesting

Comment (ronalduQualabs) [reply themes/x-mas/template.html]: Fixed

File: themes/x-mas/template.html (hunk 1 above)
Comment (ronalduQualabs) [reply themes/x-mas/template.html]: Fixed

File: themes/x-mas/template.html (hunk 2 above)
Comment (luwes) [reply themes/x-mas/template.html]: thank you! ❤️

**Pull request merged**